include/dao/dao.js

```diff
@@ -809,7 +809,7 @@ module.exports = function DAOModule(pb) {
      * @param {String} oid Object Id String
      * @return {Object}    ObjectID object
      */
-    DAO.getObjectID = function(oid) {
+    DAO.getObjectId = function(oid) {
         pb.log.warn('DAO: getObjectID is deprecated. Use getObjectId instead');
         return DAO.getObjectId(oid);
     };
```

Comment:
This is the deprecated function.  It must remain cased ```DAO.getObjectID```.  There is already a function cased the appropriate manner.